src/lem/store.rs

```diff
@@ -375,7 +375,7 @@ impl<F: LurkField> Store<F> {
 }
 
 impl<F: LurkField> Ptr<F> {
-    pub fn to_string(self, store: &Store<F>) -> String {
+    pub fn dgb_display(self, store: &Store<F>) -> String {
```

Comment:
Is this supposed to be `dbg_display`? If so, you've got 'dgb' everywhere. If not, wtf is dgb?